classes/Log/LegacyLogger.php

```diff
@@ -89,6 +89,10 @@ public function log($level, $message, array $context = array())
             return;
         }
 
+        if (!empty($this->fd)) {
+            fwrite($this->fd, '[' . date('d-m-y H:i:s') . '] ' . $message . PHP_EOL);
```

Comment:
Better to write date like `Y-m-d H:i:s` for syslog and other logs parser tools

Reply:
Here you go!